api/lib/application/organizations/index.js

```diff
@@ -526,6 +526,7 @@ exports.register = async (server) => {
           }),
           query: Joi.object({
             division: Joi.string().required(),
+            isFrenchDomainExtension: Joi.boolean(),
```

Comment:
`nitpick`
Stricto sensu, dans l'URL, `fr` est contenu dans le `hostname`, lui-même composé de domaines dont `fr` est le niveau le plus haut  (`topLevelDomain`)
https://en.wikipedia.org/wiki/Uniform_Resource_Identifier
https://en.wikipedia.org/wiki/Country_code_top-level_domain

Ce que tu nommes extension fait partie du domaine, donc `domainExtension` est ambigu.
Comme le le terme 'domaine de plus haut niveau (TLD)` est technique, je propose de l'utiliser.

Je suggère donc
```suggestion
            isTopLevelDomainFrench: Joi.boolean(),
```

Reply:
En faisant un test en local (localhost.fr:4200), le hostname ne contient que localhost et pas localhost.fr donc on ne pourrait pas tester la présence de la phrase france compétences en local.

Reply:
Voir https://github.com/1024pix/pix/pull/4545#discussion_r902231119, il est possible de passer un `hostname` custom dans les tests